ProcessMaker/processmaker
# Fix Template Update Button Error

## Issue & Reproduction Steps
This PR resolves an issue where the 'Update' button for updating an existing template in the "Save as Template" feature caused errors. 

The problem originated from the existing template ID not being passed into the request data, leading to difficulties in querying the existing template to be updated. Additionally, there was an unintended updating of the `media_collection` column, which caused further issues.

## Solution


- Included the existing template ID in the request data to facilitate querying of the existing template for updating.
- Prevented updating of the `media_collection` column to avoid unintended consequences.

## How to Test

1. Create a screen template and note the template name.
2. Create another screen template using the same name.
3. Select the 'Update' button.
4. Ensure that the existing template updates correctly without errors.

## Related Tickets & Packages
- [FOUR-14547](https://processmaker.atlassian.net/browse/FOUR-14547)

ci:next

## Code Review Checklist


[FOUR-14547]: https://processmaker.atlassian.net/browse/FOUR-14547?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
